.tekton/tasks/e2e-test.yaml

```diff
@@ -48,6 +48,11 @@ spec:
           secretKeyRef:
             name: github
             key: token
+      - name: QUAY_TOKEN
+        valueFrom:
+          secretKeyRef:
+            name: quay-repository
```

Comment:
in the `tekton-ci` namespace of production cluster, I think the secret is named as [quay-push-secret](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/tekton-ci/production/quay-push-secret.yaml)

Reply:
Thanks, just pushed the update.